src/conv-core/conv-rdma.C

```diff
@@ -113,7 +114,7 @@ void CmiIssueRput(NcpyOperationInfo *ncpyOpInfo) {
   ncpyDirectAckHandlerFn(ncpyOpInfo);
```

Comment:
We might want to add a comment here and by other uses of this function that it frees the ncpyOpInfo object